### ISSUE TITLE
PATCH with unitCode is not parsed correctly

### ISSUE THREAD

**Issue by jason-fox** [Fiware/context.Orion-LD#505]
I create an entity with a `unitCode` property:

```console
curl -L -X POST 'http://localhost:1026/ngsi-ld/v1/entities/' \
-H 'Content-Type: application/json' \
-H 'Link: <http://context-provider:3000/data-models/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
      "id": "urn:ngsi-ld:TemperatureSensor:001",
      "type": "TemperatureSensor",
      "category": {
            "type": "Property",
            "value": "sensor"
      },
      "temperature": {
            "type": "Property",
            "value": 25,
            "unitCode": "CEL"
      }
}'
```

I attempt to PATCH the entity:

```console
curl -L -X PATCH 'http://localhost:1026/ngsi-ld/v1/entities/urn:ngsi-ld:TemperatureSensor:001/attrs/temperature' \
-H 'Content-Type: application/json' \
-H 'Link: <http://context-provider:3000/data-models/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
    "value": 89,
    "type": "Property"
}'
```

Response states:

```json
{
    "type": "https://uri.etsi.org/ngsi-ld/errors/BadRequestData",
    "title": "The 'type' field is missing for a sub-attribute",
    "detail": "https://uri.etsi.org/ngsi-ld/unitCode"
}
```

However according to section **5.2.5 Property** of the ETSI specification the following elements:

-  `observedAt` **string** DateTime (clause 4.6.3) 0..1 Timestamp. See clause 4.8
-  `unitCode` **string** As mandated by [15] 0..1 Property Value's unit code
-  `datasetId` **URI** 0.1 It allows identifying a set or group of property values

Do not have the data type **Property** and therefore do not require `type` sub-attribute.

```
"git_hash": "db891a72a531ef4cc037caed5b04f8fe71da6814",
"compile_time": "Thu Jul 30 13:13:23 UTC 2020"
```

**Comment by jason-fox** [comment]
The `@context` used in the example can be found [here](https://raw.githubusercontent.com/FIWARE/tutorials.NGSI-LD/master/app/public/data-models/ngsi-context.jsonld)

**Comment by kzangeli** [comment]
Fixed in PR #514

**Comment by jason-fox** [comment]
Verified eddd37530fb7cb748ad582b5c20ea4327accd558
